xarray/core/ops.py

```diff
@@ -243,6 +248,42 @@ def rolling_count(rolling):
 
     return rolling_count.where(enough_periods)
 
+
+def dask_rolling_wrapper(moving_func, a, window, min_count=None, axis=-1):
+    '''wrapper to apply bottleneck moving window funcs on dask arrays'''
+    # inputs for ghost
+    if axis < 0:
+        axis = a.ndim + axis
+    depth = {d: 0 for d in range(a.ndim)}
+    depth[axis] = window - 1
+    boundary = {d: np.nan for d in range(a.ndim)}
+    # create ghosted arrays
+    ag = da.ghost.ghost(a, depth=depth, boundary=boundary)
+    # apply rolling func
+    out = ag.map_blocks(moving_func, window, min_count=min_count,
+                        axis=axis, dtype=a.dtype)
+    # trim array
+    result = da.ghost.trim_internal(out, depth)
+    return result
+
+
+def dask_rolling_wrapper_without_min_count(moving_func, a, window, axis=-1):
```

Comment:
Can't we use the default parameter value rather than writing this twice?

Reply:
If we update the minimum support bottleneck version to 1.1, we can do away with this special case (also present for the non-dask case) all together.